CMakeLists.txt

```diff
@@ -578,6 +578,8 @@ if (WIN32 OR MINGW)
   add_compile_definitions (_BIND_TO_CURRENT_VCLIBS_VERSION=1 _CRT_SECURE_NO_WARNINGS _CONSOLE)
 endif ()
 
+option (HDF5_MINGW_STATIC_LIBGCC "Statically link libgcc/libstdc++" ON)
```

Comment:
New options need to be added to INSTALL_CMake.txt and RELEASE.tx files.

Reply:
ok, done